From 09b0ab9b43846b464170941fd0eaec90b588d610 Mon Sep 17 00:00:00 2001
From: Chris Wilson <chris@chris-wilson.co.uk>
Date: Wed, 18 Feb 2015 19:16:29 +0000
Subject: uxa/dri: Drop dependency on stdbool

Copying the bool is_level() without including stdbool.h causes it to not
compile on some machines. Just convert back to the old faithful Bool.

Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>

diff --git a/src/uxa/intel_dri.c b/src/uxa/intel_dri.c
index 32622ad..524826d 100644
--- a/src/uxa/intel_dri.c
+++ b/src/uxa/intel_dri.c
@@ -1488,37 +1488,37 @@ namecmp(const char *s1, const char *s2)
 	return c1 - c2;
 }
 
-static bool is_level(const char **str)
+static Bool is_level(const char **str)
 {
 	const char *s = *str;
 	char *end;
 	unsigned val;
 
 	if (s == NULL || *s == '\0')
-		return true;
+		return TRUE;
 
 	if (namecmp(s, "on") == 0)
-		return true;
+		return TRUE;
 	if (namecmp(s, "true") == 0)
-		return true;
+		return TRUE;
 	if (namecmp(s, "yes") == 0)
-		return true;
+		return TRUE;
 
 	if (namecmp(s, "0") == 0)
-		return true;
+		return TRUE;
 	if (namecmp(s, "off") == 0)
-		return true;
+		return TRUE;
 	if (namecmp(s, "false") == 0)
-		return true;
+		return TRUE;
 	if (namecmp(s, "no") == 0)
-		return true;
+		return TRUE;
 
 	val = strtoul(s, &end, 0);
 	if (val && *end == '\0')
-		return true;
+		return TRUE;
 	if (val && *end == ':')
 		*str = end + 1;
-	return false;
+	return FALSE;
 }
 
 static const char *options_get_dri(intel_screen_private *intel)
-- 
cgit v0.10.2

